projects/plugins/jetpack/extensions/blocks/dialogue/edit.js

```diff
@@ -127,34 +122,26 @@ export default function DialogueEdit( {
 		}
 
 		// Check if the participant has been removed from Conversation.
-		if ( currentParticipant ) {
+		if ( conversationParticipant ) {
 			return;
 		}
 
 		// Set first participant as default.
-		setAttributes( { participantSlug: participants[ 0 ].participantSlug } );
-	}, [ participants, currentParticipant, setAttributes ] );
+		setAttributes( { participant: participants[ 0 ] } );
+	}, [ participants, conversationParticipant, setAttributes ] );
 
 	function hasStyle( style ) {
-		return currentParticipant?.[ style ];
+		return participant?.[ style ];
 	}
 
 	function toggleParticipantStyle( style ) {
 		conversationBridge.updateParticipants( {
-			participantSlug,
-			[ style ]: ! currentParticipant[ style ],
-		} );
-	}
-
-	function getParticipantLabelClass() {
-		return classnames( `${ baseClassName }__participant`, {
-			[ 'has-bold-style' ]: currentParticipant?.hasBoldStyle,
-			[ 'has-italic-style' ]: currentParticipant?.hasItalicStyle,
-			[ 'has-uppercase-style' ]: currentParticipant?.hasUppercaseStyle,
```

Comment:
`getParticipantLabelClass` is extracted into a shared util file so it can be used in the save function, as well.